OracleDatabase/RAC/OracleDNSServer/dockerfiles/latest/named.localhost

```diff
@@ -0,0 +1,10 @@
+$TTL 1D
```

Comment:
The `named` package should install this file. You should not provide it yourself.